src/components/BookCard/BookCard.jsx

```diff
@@ -12,8 +12,13 @@ export default function BookCard({
 			<img src={`../images/${coverImg}`} />
 			<div className="card-content">
 				<p className="card-title"> {title}</p>
-				<p className="card-author">
-					By {authors} ({publishDate})
+				<p className="book-author">
+					{Array.isArray(authors)
+						? `By ${authors.join(', ')}`
+						: `By ${authors || 'Unknown Author'}`}{' '}
```

Comment:
I think we can delete the extra space `{' '}` at the end of this line.

Reply:
i think they appear after format on save but i'll check it out